src/components/HelpDrawer/HelpContent/StokesAnalysisHelpContent.tsx

```diff
@@ -20,11 +20,13 @@ export const STOKES_ANALYSIS_HELP_CONTENT = (
         </ul>
         <p>All these plots are inter-linked so that when zooming profiles, data in the visible range will be highlighted in the scatter plot, and vice versa.</p>
         <h3 id="images">Image dropdown menu</h3>
-        <p>The image dropdown defaults to &quot;Active&quot; image which means the current image in the image viewer.</p>
+        <p>
+            The <code>Image</code> dropdown defaults to &quot;Active&quot; image which means the current image in the image viewer if it is single panel mode. If it is multi-panel mode, the active image is highlighted with a red box.
+        </p>
         <h3 id="regions">Region dropdown menu</h3>
         <p>
-            The region dropdown defaults to &quot;Active&quot; region which means a selected region in the image viewer. You can select a region by clicking one on the image viewer, or by clicking a region entry on the region list widget.
-            Stokes profile plot of the selected region will be updated accordingly. If no region is selected, &quot;Active&quot; region defaults to cursor.
+            The <code>Region</code> dropdown defaults to &quot;Active&quot; region which means a selected region in the image viewer. You can select a region by clicking one on the image viewer, or by clicking a region entry on the region
```

Comment:
"the region selected in the image viewer"

"clicking on one in the image viewer"

"on a region entry in the..."